app.js

```diff
@@ -27,7 +27,9 @@ function checkGateways (gateways) {
   let checked = 0
   gateways.forEach((gateway) => {
     const gatewayAndHash = gateway.replace(':hash', hashToTest)
-    fetch(gatewayAndHash)
+    // opt-out from gateway redirects done by browser extension
+    const testUrl = gatewayAndHash + '#x-ipfs-companion-no-redirect'
```

Comment:
@VictorBjelkholm  FYI this feature shipped to Beta in [ipfs-companion/releases/tag/v2.4.0.10120](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.4.0.10120)